tests/serializers/test_definitions.py

```diff
@@ -13,7 +11,7 @@ def test_custom_ser():
         )
     )
     assert s.to_python([1, 2, 3]) == ['1', '2', '3']
-    assert plain_repr(s).endswith('slots=[])')
+    # assert plain_repr(s).endswith('slots=[])')
```

Comment:
What to do with these tests? Not sure.

Reply:
this is testing that slots weren't used, I guess because we previously identified that the validator was only used once, so inlined it.